src/main/java/org/gridsuite/network/map/dto/mapper/VoltageLevelInfosMapper.java

```diff
@@ -132,18 +128,8 @@ protected static VoltageLevelTabInfos toTabInfos(Identifiable<?> identifiable) {
                 .nominalVoltage(voltageLevel.getNominalV())
                 .lowVoltageLimit(nullIfNan(voltageLevel.getLowVoltageLimit()))
                 .highVoltageLimit(nullIfNan(voltageLevel.getHighVoltageLimit()));
-
-        IdentifiableShortCircuit<VoltageLevel> identifiableShortCircuit = voltageLevel.getExtension(IdentifiableShortCircuit.class);
-        if (identifiableShortCircuit != null) {
-            Double ipMin = nullIfNan(identifiableShortCircuit.getIpMin());
-            if (ipMin != null) {
-                builder.ipMin(ipMin / 1000.);  // to get value in kA : value is in A in iidm
-            }
-            Double ipMax = nullIfNan(identifiableShortCircuit.getIpMax());
-            if (ipMax != null) {
-                builder.ipMax(ipMax / 1000.);  // to get value in kA : value is in A in iidm
-            }
-        }
+        // TODO conversion frontend to KA
```

Comment:
this TODO is for the front, not the back

Reply:
I mean that the unitt conversion on backend and it will be done on front end now ( I left the comment to explain why I changed it)